.changes/unreleased/tinymce-TINY-10675-2024-02-25.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Advcode's styles wasn't working as expected.
```

Comment:
This change was never released so we shouldn't have a changelog item since the customer never saw the broken state.

- [ ] Remove changelog item.

Reply:
done